### PR TITLE
Fix the bulk "move to collection" action

### LINKED ISSUE #5017: Can't move questions using the bulk move action
Trying to move a question from the Everything Else section into a collection from the Questions page **using the bulk-move control at the top of the list of questions** appears to do something, but then on a reload of the page or by navigating away and back again, the question does not in fact actually get moved.

### PR DESCRIPTION
**Pull request by attekei**
Fixes #5017

### CODE REVIEW

File: frontend/src/metabase/questions/questions.js [hunk 1]
```diff
@@ -176,7 +176,7 @@ export const setCollection = createThunkAction(SET_COLLECTION, (cardId, collecti
                 )));
                 MetabaseAnalytics.trackEvent("Questions", "Bulk Move to Collection");
             }
-            selected.map(item => dispatch(setCollection(item.id, collectionId)));
+            selected.map(item => dispatch(setCollection(item.id, { id: collectionId })));
```
Comment (salsakran) [inline frontend/src/metabase/questions/questions.js]: Ick.

Is there a reason for that method signature? 
`f(id, {id: <id>})`

Comment (attekei) [inline frontend/src/metabase/questions/questions.js]: Seems that `collection` may also contain `slug` property but that doesn't make it much cooler.

Comment (salsakran) [inline frontend/src/metabase/questions/questions.js]: so the idea is that you can move it into a collection by either the id or the slug?

**Review by mazameli — APPROVED** [review]
Behavior is fixed. 👍

**Review by kdoh — APPROVED** [review]
Agree with @salsakran r.e. that method signature but otherwise LGTM.

**Review by tlrobinson — APPROVED** [review]

**Comment by attekei** [comment]
Merging this now. We talked earlier (when working on #4953) about how it's hard to use question actions outside the questions list, so the actions definitely need some overhaul. I think that it should be addressed separately (I don't want to change the signature of functions without looking at bigger picture).

**Pull request merged**